workflows/subworkflows/cmsearch-condition.cwl

```diff
@@ -0,0 +1,64 @@
+cwlVersion: v1.2.0-dev2
+class: Workflow
+$namespaces:
+  edam: 'http://edamontology.org/'
+  s: 'http://schema.org/'
+label: Identifies non-coding RNAs using Rfams covariance models
+
+requirements:
+  SubworkflowFeatureRequirement: {}
+  MultipleInputFeatureRequirement: {}
+  InlineJavascriptRequirement: {}
```

Comment:
I think you don't need InlineJavascriptRequirement nor ScatterFeatureRequirement?